README.md

```diff
@@ -34,6 +34,7 @@ As API framework, we use [Feathers](http://feathersjs.com).
     ```
```

Comment:
Step 2 from the Getting started section can be removed, I think. Nodemon should be installed as a dependecy by the command in step 3.